feathr_project/feathr/api/app/backends/purview_backend/purview_backend.py

```diff
@@ -27,6 +27,10 @@ def ListProjects(self):
         raw_result = registry_client._list_registered_entities_with_details(entity_type=[TYPEDEF_FEATHR_PROJECT])
```

Comment:
Shall we also change the `self.default_project_name = "feathr_github_ci_synapse"` to make it configurable?